apps/content-publishing-worker/src/worker.module.ts

```diff
@@ -58,6 +59,7 @@ import { ContentPublishingQueues as QueueConstants } from '#types/constants';
       ignoreErrors: false,
     }),
     ScheduleModule.forRoot(),
+    IPFSStorageModule,
```

Comment:
This probably doesn't need to be imported here, since WorkerModule doesn't directly instantiate any providers, and the sub-modules that require it all import it themselves anyway.

Although, probably worth experimenting--I think this might be a good use case for making this a global module; that way you only have to import it once here & all sub-modules can use its exported services. The way it's set up right now, I think each module that imports it is getting its own instance, which seems unnecessary. That's the pitfall of dependency injection: it makes it a bit too easy to unintentionally instantiate multiple copies of things.

Or maybe I'm wrong; but worth checking.

Reply:
based on the experiments it looks like we only have 1 instance by default